operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -468,9 +478,26 @@ server:
 {{- end }}
   log_level: info
 storage_config:
+{{- range $ship, $_ := $shippersDic }}
+{{- if eq $ship "boltdb" }}
   boltdb_shipper:
-    active_index_directory: {{ .StorageDirectory }}/index
-    cache_location: {{ .StorageDirectory }}/index_cache
+    active_index_directory: {{ $.StorageDirectory }}/index
+    cache_location: {{ $.StorageDirectory }}/index_cache
+{{- end }}
+{{- if eq $ship "tsdb" }}
+  tsdb_shipper:
+    active_index_directory: {{ $.StorageDirectory }}/tsdb-index
+    cache_location: {{ $.StorageDirectory }}/tsdb-cache
```

Comment:
Based on the [migration document](https://grafana.com/docs/loki/v2.9.x/operations/storage/tsdb/#index-caching-not-required), TSDB doesn't require index caching.
I think we can safely remove this line from the config (and the unit tests above)
```suggestion
```

Reply:
I think there they are refering to `index_queries_cache_config` here is a message from Pery when I asked for clarification on that part.

> If you look here https://grafana.com/docs/loki/latest/operations/caching/ you will notice that Loki has several caching sections, one of this is `index_queries_cache_config` which we don't support anyway. We have activated only the results_cache and the chunk_cache, because our LokiStacks are quite small and spending extra 500MB per index-gateway for caching index queries is not worth it.